xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -100,7 +96,7 @@ CGUIMediaWindow::CGUIMediaWindow(int id, const char *xmlFile)
   m_iLastControl = -1;
   m_canFilterAdvanced = false;
 
-  m_guiState.reset(CGUIViewState::GetViewState(GetID(), *m_vecItems));
+  m_guiState.reset(CGUIViewState::GetViewState(CGUIControl::GetID(), *m_vecItems));
```

Comment:
`CGUIControl::GetID()` is virtual and could be overridden by any of the classes in the inheritance hierarchy of `CGUIMediaWindow`. Why should we explicitly bypass that?